OFFSPEC_runs_46992-4/README.md

```diff
@@ -1,4 +1,3 @@
 # Neutron Reflectivity Reference Data Set: OFFSPEC
 
-This data set represents a published study of the determination of a thin-film structure using Neutron Reflectivity: *ACS Appl. Mater. Interfaces* 2019, 11, 14, 13803–13811 (https://doi.org/10.1021/acsami.9b02700). The raw data is available at https://data.isis.stfc.ac.uk/doi/INVESTIGATION/84794745/. The [raw data nexus file](https://github.com/DAaaS-reference-data/Reflectivity/blob/main/OFFSPEC_runs_46992-4/Runs_46992_46993_46994_polymer_on_Si.nxs), the [reduced data](https://github.com/DAaaS-reference-data/Reflectivity/blob/main/OFFSPEC_runs_46992-4/Runs_46992_46993_46994_polymer_on_Si.dat) and the [reduction script](https://github.com/DAaaS-reference-data/Reflectivity/blob/main/OFFSPEC_runs_46992-4/RB1720415_run_46992.py) to obtain one from the other are all included in this directory.
-
+This data set represents a published study of the determination of a thin-film structure using Neutron Reflectivity: *ACS Appl. Mater. Interfaces* 2019, 11, 14, 13803–13811 (https://doi.org/10.1021/acsami.9b02700). The raw data is available at https://data.isis.stfc.ac.uk/doi/INVESTIGATION/84794745/. The raw data nexus files, as well as the original reduced data (in .csv format) are both in ../OFFSPEC_runs_46992-4/data_files/. The main directory contains a Jupyter notebook explaining how the data is reduced.
```

Comment:
I think it would be useful to have some links or filenames or file extensions to make it really explicit which data files correspond to the raw and reduced data. In addition, the "../OFFSPEC_runs_46992-4/data_files/" location could be simplified to "./data_files" or similar I think.